examples/mapdl_tyre_performance/interrupt_task.py

```diff
@@ -64,19 +64,28 @@ def interrupt_running_task(client, project_id) -> Project:
     resp = client.session.get(command_defs_url)
     command_defs = resp.json()["task_command_definitions"]
     if not command_defs:
-        log.info(f"No command definitions found")
+        log.error(f"No command definitions found")
```

Comment:
```suggestion
        log.error(f"No command definitions are found")
```
Messages must be complete, with a subject AND a verb.